scripts/clean-grant-data

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+'use strict';
+const fs = require('fs');
+const es = require('event-stream');
+const moment = require('moment');
+const _ = require('lodash');
+const split = require('split');
+
+const argv = require('yargs')
+    .alias('f', 'file')
+    .describe('f', 'The path to a JSON file to clean')
+    .alias('o', 'output')
+    .describe('b', 'The path to the JSON file to output')
+    .help('h')
+    .alias('h', 'help').argv;
+
+// Turn date fields into actual Date objects
+const parseDates = data => {
+    const dateFields = [
+        'Award Date',
+        'Planned Dates:Start Date',
+        'Planned Dates:End Date'
+    ];
+    dateFields.forEach(dateField => {
+        if (data[dateField]) {
+            let d = moment(data[dateField].trim(), 'YYYY-MM-DD');
+            data[dateField] = d.toDate();
```

Comment:
Noting that we probably want to output this as a date _type_ for mongo. Something like this: https://stackoverflow.com/questions/33107278/import-a-dateisodate-into-mongodb-using-mongoimport

Reply:
Good call. Done:
![image](https://user-images.githubusercontent.com/394376/45704481-e6561d00-bb6e-11e8-8533-86fa8fcee984.png)

Reply:
Compared to before:

![image](https://user-images.githubusercontent.com/394376/45704562-22897d80-bb6f-11e8-8b86-3d212d6d67df.png)